uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -72,6 +72,30 @@ define([
       }
 
       /**
+       * Load file extensions which are supported for upload
+       *
+       * @return {Promise}
+       */
+      function loadSupportedFileTypes () {
+        return OptionGroup.valuesOf('safe_file_extension')
+          .then(function (extensions) {
+            var allowedMimeTypes = {};
+            extensions.map(function (ext) {
+              allowedMimeTypes[ext.label] = sharedSettings.fileUploader.mimeTypesMap[ext.label];
```

Comment:
What happens if mimeTypesMap does not have any extension (.7z for example)? Shouldn't there be a some kind of fallback?